crates/re_types/src/archetypes/depth_image_ext.rs

```diff
@@ -12,7 +12,10 @@ impl DepthImage {
     /// for treating as an image.
     ///
     /// This is useful for constructing an [`DepthImage`] from an ndarray.
-    pub fn try_from<T: TryInto<TensorData>>(data: T) -> Result<Self, ImageConstructionError<T>> {
+    pub fn try_from<T: TryInto<TensorData>>(data: T) -> Result<Self, ImageConstructionError<T>>
+    where
+        <T as TryInto<TensorData>>::Error: std::fmt::Debug,
```

Comment:
leftover from debugging?

Reply:
Required for:
```
pub enum ImageConstructionError<T: TryInto<TensorData>>
where
    T::Error: std::fmt::Debug,
{
    #[error("Could not convert source to TensorData: {0:?}")]
    TensorDataConversion(T::Error),
```
to be able to work.

I could switch to `{0}` and `std::fmt::Display` but assumed Debug was more likely to be implemented.